pkg/scheduler/scheduler.go

```diff
@@ -744,9 +735,11 @@ type Reconciler struct {
 	implicit chan struct{}   // send an empty struct to trigger implicit reconciliation
 	done     <-chan struct{} // close this when you want the reconciler to exit
 	cooldown time.Duration
+	explicitReconciliationAbortTimeout time.Duration
 }
 
-func newReconciler(doer proc.Doer, action ReconcilerAction, cooldown time.Duration, done <-chan struct{}) *Reconciler {
+func newReconciler(doer proc.Doer, action ReconcilerAction, cooldown time.Duration,
```

Comment:
also, more idiomatic to declare, for example, `f(a,b int)` instead of `f(a int, b int)` .. w/ respect to the time.Duration params